_includes/sidenav.html

```diff
@@ -24,16 +23,17 @@
 
 <nav class="nav-secondary expandable {{ additional_classes }}">
     <h1 class="u-visually-hidden">
```

Comment:
I was curious about this too, since I've seen them listed as an error when testing. Figured it was the plugin being out of date.